test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -798,13 +802,22 @@ void ProtoConverter::visit(CopyFunc const& _x)
 	case CopyFunc::DATA:
 		m_output << "datacopy";
 		break;
+	case CopyFunc::MEMORY:
+		m_output << "mcopy";
 	}
 	m_output << "(";
 	m_output << "mod(";
 	visit(_x.target());
 	m_output << ", " << to_string(s_maxMemory - s_maxSize) << ")";
 	m_output << ", ";
-	visit(_x.source());
+	if (type == CopyFunc::MEMORY)
+	{
+		m_output << "mod(";
+		visit(_x.source());
+		m_output << ", " << to_string(s_maxMemory - s_maxSize) << ")";
```

Comment:
Looks like this assumes that the gap between `s_maxMemory` and `s_maxSize` is large enough that we have a sensible range of source/target addresses to test. Also that `s_maxMemory` is not smaller than `s_maxSize`. Since they're defined elsewhere it would be easy for someone to violate that assumption and not even notice. Would be nice to have a comment about that near their definitions. An assert here checking for some minimum value would not hurt either.

Reply:
Added comment. I would like to add an assert but may be in another PR, since adding it here would be a hack and doing it cleanly would require some code refactoring. Don't want to delay fuzzer support.